ui/plugin-system/src/components/PluginEditor/plugin-editor-api.ts

```diff
@@ -29,6 +29,7 @@ export interface PluginEditorProps extends Omit<BoxProps, OmittedMuiProps> {
   pluginKindLabel: string;
   value: Definition<UnknownSpec>;
   isReadonly?: boolean;
+  isExplore?: boolean;
```

Comment:
I’m confused why isExplore prop is added in so many places, can explore instead use the lower level building blocks directly? We can export them out if any are missing but it would give more control than using the higher level components that are more for dashboards and when the spec needs to be saved (explore is view-only right?)

Reply:
We thought about that, but this will require duplicate many components we're already have just because we want a different render style. Don't you think?